src/main/webapp/src/app/admin-page/admin-page.component.ts

```diff
@@ -159,6 +165,63 @@ export class AdminPageComponent implements OnInit {
     });
   }
 
+  /**
+   * Posts a plant to a garden.
+   *
+   * Performs POST: /garden/{{gardenProfile.id}}/plant-list, with
+   * body plant.
+   *
+   * @param plant the plant to post.
+   * @return the http response.
+   */
+  postPlantToGarden(plant: Plant): Observable<HttpResponse<string>> {
+    console.log(JSON.stringify(plant));
+    return this.httpClient.post<string>(
+      '/garden/' + this.gardenProfile.id + '/plant-list',
+      plant,
+      {
+        observe: 'response',
+        responseType: 'json',
+      }
+    );
+  }
+
+  /**
+   * Deletes a plant from a garden.
+   *
+   * Performs GET: /garden/{{gardenProfile.id}}/plant-list/{id}
+   *
+   * @param id the plant id to delete.
+   * @return the http response.
+   */
+  deletePlant(id: string): Observable<HttpResponse<string>> {
+    return this.httpClient.delete<string>(
+      '/garden/' + this.gardenProfile.id + '/plant-list/' + id,
+      {
+        observe: 'response',
+        responseType: 'json',
+      }
+    );
+  }
+
+  /**
+   * Deletes a user from a garden.
+   *
+   * Performs GET: /garden/{{gardenProfile.id}}/user-list/{id}
+   *
+   * @param id the user id to delete.
+   * @return the http response.
+   */
+  deleteUser(id: string): Observable<HttpResponse<string>> {
```

Comment:
It could be helpful to leave a `TODO` to guard `deletePlant` and `deleteUser` once you are able to verify login status